README.md

```diff
@@ -17,12 +17,6 @@ See: http://aem.live/docs/sidekick-generate-variations
     - `aio console workspace select`
 - Populate the `.env` file in the project root and fill it as shown [below](#env)
 
-### Local Development
-
-- `npm start` to start your local Dev server
```

Comment:
@irenelagno , Should we delete https://github.com/adobe/aem-genai-assistant/blob/SITES-21521/package.json#L7 as well ?

Reply:
I will do with my next PR! good catch

Reply:
fixed